docs/Module/Import-WinModule.md

```diff
@@ -23,9 +23,18 @@ Import-WinModule [[-Name] <String[]>] [-Exclude <String[]>] [-ComputerName <Stri
 
 This command allows you to import proxy modules from a local or remote session.
 These proxy modules will allow you to invoke cmdlets that are not directly supported in this version of PowerShell.
+
 There are commands in the Windows PowerShell core modules that don't exist natively in PowerShell Core.
 If these modules are imported, proxies will only be created for the missing commands.
 Commands that already exist in PowerShell core will not be overridden.
+These modules are:
+
+- Microsoft.PowerShell.Management
+- Microsoft.PowerShell.Utility
+- Microsoft.PowerShell.Security
+- Microsoft.PowerShell.Diagnostics
+
+Commands from much of the other modules can be fully imported.
```

Comment:
I think this sentence actually makes things s bit more confusing. It could probably be reworded for greater clarity, but I think the rest is explanatory enough without it.